src/multi.rs

```diff
@@ -203,6 +208,32 @@ impl MultiState {
         extra_lines: Option<Vec<String>>,
         now: Instant,
     ) -> io::Result<()> {
+        // Reap all consecutive 'zombie' progress bars from head of elements
+        let mut zombies = vec![];
```

Comment:
So this will allocate any time there are zombies. Should we store it in the `MultiProgressState` so we don't have to reallocate on every draw with zombies, or do we think draws with zombies are relatively rare?

Reply:
Depends on the usage pattern. If users are creating and dropping pbs a lot, then it's going to allocate a lot. Seems like a premature optimization to stuff it into `MultiState` for now - would you be ok with us waiting for someone to complain?